api/client/volume/cmd.go

```diff
@@ -0,0 +1,33 @@
+package volume
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/docker/docker/api/client"
+	"github.com/docker/docker/cli"
+)
+
+// NewVolumeCommand returns a cobra command for `volume` subcommands
+func NewVolumeCommand(dockerCli *client.DockerCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "volume",
+		Short: "Manage Docker volumes",
+		// TODO: remove once cobra is patched to handle this
```

Comment:
The comment doesn't say much to what cobra should/will handle (by reading the code I'm not sure I'm guessing it right).